Mathlib/Analysis/NormedSpace/Multilinear/Basic.lean

```diff
@@ -1199,14 +1216,49 @@ theorem compContinuousLinearMapL_apply (g : ContinuousMultilinearMap 𝕜 E₁ G
   rfl
 #align continuous_multilinear_map.comp_continuous_linear_mapL_apply ContinuousMultilinearMap.compContinuousLinearMapL_apply
 
-variable (G)
-
+variable (G) in
 theorem norm_compContinuousLinearMapL_le (f : ∀ i, E i →L[𝕜] E₁ i) :
     ‖compContinuousLinearMapL (G := G) f‖ ≤ ∏ i, ‖f i‖ :=
   LinearMap.mkContinuous_norm_le _ (prod_nonneg fun _ _ => norm_nonneg _) _
 #align continuous_multilinear_map.norm_comp_continuous_linear_mapL_le ContinuousMultilinearMap.norm_compContinuousLinearMapL_le
 
-variable (𝕜 E E₁)
+/-- `ContinuousMultilinearMap.compContinuousLinearMap` as a bundled continuous linear map.
+This implementation fixes `g : ContinuousMultilinearMap 𝕜 E₁ G`.
+
+Actually, the map is linear in `g`,
+see `ContinuousMultilinearMap.compContinuousLinearMapContinuousMultilinear`.
+
+For a version fixing `f` and varying `g`, see `compContinuousLinearMapL`. -/
+def compContinuousLinearMapLRight (g : ContinuousMultilinearMap 𝕜 E₁ G) :
+    ContinuousMultilinearMap 𝕜 (fun i ↦ E i →L[𝕜] E₁ i) (ContinuousMultilinearMap 𝕜 E G) :=
+  MultilinearMap.mkContinuous
+    { toFun := fun f => g.compContinuousLinearMap f
+      map_add' := by
+        intro h f i f₁ f₂
+        ext x
+        change (g fun j ↦ update f i (f₁ + f₂) j <| x j) =
+            (g fun j ↦ update f i f₁ j <| x j) + g fun j ↦ update f i f₂ j (x j)
```

Comment:
My understanding is that `change` is prone to breakage if definitions change, so using the relevant theorems is more appropriate. If this is wrong, please let me know!
```suggestion
        simp only [compContinuousLinearMap_apply, add_apply]
```